packages/angular/projects/ui-angular/src/lib/components/authenticator/components/reset-password/reset-password.component.html

```diff
@@ -1,14 +1,15 @@
 <div data-amplify-container>
+  <amplify-slot name="reset-password-header">
+    <h3 class="amplify-heading">{{ this.headerText }}</h3>
+  </amplify-slot>
```

Comment:
The problem with moving this outside is that it loses the padding around the form.

This is better for customization (because breaking out of padding is hard!), but what is the best answer in the Angular templates for having this padding?

> <img width="610" alt="Screen Shot 2021-11-16 at 10 27 01 AM" src="https://user-images.githubusercontent.com/15182/142025560-6f4d6725-4cfc-4457-a5f6-6a8257e0e79c.png">

Reply:
Let's just style it?